modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/PlayerBottomSheet.kt

```diff
@@ -134,14 +134,14 @@ class PlayerBottomSheet @JvmOverloads constructor(context: Context, attrs: Attri
         binding.miniPlayer.setPlaybackState(playbackState)
     }
 
-    fun setUpNext(upNext: UpNextQueue.State, theme: Theme) {
+    fun setUpNext(upNext: UpNextQueue.State, theme: Theme, shouldAnimateOnAttach: Boolean) {
         binding.miniPlayer.setUpNext(upNext, theme)
 
         // only show the mini player when an episode is loaded
         if (upNext is UpNextQueue.State.Loaded) {
             if ((isHidden() || !hasLoadedFirstTime)) {
                 show()
-                if (!shouldPlayerOpenOnAttach) {
+                if (!shouldPlayerOpenOnAttach && shouldAnimateOnAttach) {
```

Comment:
This is preventing `listener?.onMiniPlayerVisible()` from getting called on line 148 after changing themes. As a result, the padding isn't being updated to handle the mini player being visible, and we're losing content under the miniplayer. I was able to see this by following these steps:

1. Open the app and go to Profile tab -> Gear Icon
2. Scroll to the bottom of the screen and observe that the last two settings are "Privacy" followed by "About"
3. Tap on Appearance and change the theme
4. Tap back and scroll to the bottom
5. Observe that Privacy is now the last visible setting (because "About" is now hidden behind the miniplayer).

https://user-images.githubusercontent.com/4656348/199075257-fcca4e34-0277-4626-92bc-4056d0a82cfb.mp4

Similarly, it seems like we should also be updating the `hasLoadedFirstTime` field here, although I'm not sure that not updating it is actually causing any problems currently.